test/files/pos/t11020.scala

```diff
@@ -0,0 +1,7 @@
+// ASF crash related to BTS bug for singleton types
```

Comment:
https://www.youtube.com/watch?v=uUV3KvnvT-w&feature=youtu.be&t=1m37s

Reply:
How about I just add this link to clarify? 😁

Reply:
Ah you meant the other comment. I just zoned out again while reading it. I'll see if I can make sense of it.

Reply:
For the innocent onlooker: ASF == asSeenFrom, BTS == BaseTypeSeq. I'll update this comment too.